helm/superset/Chart.yaml

```diff
@@ -22,7 +22,7 @@ maintainers:
   - name: craig-rueda
     email: craig@craigrueda.com
     url: https://github.com/craig-rueda
-version: 0.16.0
+version: 0.17.0
```

Comment:
@satishblotout We can add a quote here like `"0.17.0"`

Reply:
done.